rskj-core/src/jmh/java/co/rsk/jmh/web3/BenchmarkWeb3.java

```diff
@@ -46,13 +46,13 @@ public class BenchmarkWeb3 {
 
     @Benchmark
     @Timeout(time = 60)
-    public void ethCallForSpecificBlock(TransactionPlan plan) throws BenchmarkWeb3Exception {
+    public void ethCallForSpecificBlock(EthCallPlan plan) throws BenchmarkWeb3Exception {
```

Comment:
`EthCallPlan` related methods need a local wallet for contract pre-creation, therefore they should be part of `BenchmarkLocalWalletWeb3` and not `BenchmarkWeb3`, please move them.